spec/fixtures/application/1.0/application.json

```diff
@@ -146,6 +146,63 @@
           "metadata": { "payee_name": "Willy Wonka", "payee_relationship_to_client": "Her Eccentric Uncle" }
         }
       ]
+    },
+    "capital_details": {
+      "has_premium_bonds": "no",
+      "premium_bonds_total_value": null,
+      "premium_bonds_holder_number": null,
+      "will_benefit_from_trust_fund": "no",
+      "trust_fund_amount_held": null,
+      "yearly_dividend": null,
+      "savings": [
+        {
+          "saving_type": "bank",
+          "provider_name": "Halifax",
+          "sort_code": "123456",
+          "account_number": "12345678",
+          "account_balance": 200,
+          "is_overdrawn": "yes",
+          "are_wages_paid_into_account": "yes",
+          "ownership_type": "applicant"
+        }
+      ],
+      "national_savings_certificates": [
+        {
+          "holder_number": "A123",
+          "certificate_number": "B456",
+          "value": 20010,
+          "ownership_type": "partner"
+        }
+      ],
+      "investments": [
+        {
+          "investment_type": "unit_trust",
+          "description": "Details of investment",
+          "value": 200,
+          "ownership_type": "partner"
+        }
+      ],
+      "properties": [
+        {
+          "property_type": "residential",
+          "house_type": "custom",
+          "custom_house_type": "custom house type",
+          "size_in_acres": 100,
+          "usage": "usage details",
+          "bedrooms": 2,
+          "value": 200000,
```

Comment:
Just an FYI, these are all in pence, if you did want them to be realisticish - this would be be a £2000 house

Reply:
Good point - I've made those changes